guides/v2.1/howdoi/patch_from_github.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+group: howdoi
+title: Create a GitHub patch for a Composer installation
+version: 2.2
+github_link: howdoi/patch_from_github.md
+functional_areas:
+  - Install
+  - System
+  - Setup
+---
+
+Sometimes it takes a while for us to include a bug fix made on GitHub in a Magento 2 Composer release. In the meantime, you can create a patch from GitHub and use the `composer-patches` plugin to apply it to your Composer-based Magento 2 installation.
+
+<div class="bs-callout bs-callout-warning" markdown="1">
+Be careful applying an unreleased patch. Do comprehensive testing before deploying it.
+</div>
+
+## Create a patch
+
+To create a patch file from a GitHub commit or pull request, append `.patch` to the url, [https://github.com/magento/magento2/commit/2d31571f1bacd11aa2ec795180abf682e0e9aede.patch](https://github.com/magento/magento2/commit/2d31571f1bacd11aa2ec795180abf682e0e9aede.patch).
+
+<div class="bs-callout bs-callout-info" markdown="1">
+You must change the paths in the patch file to correspond to the `vendor/***` directories.
+</div>
+
+## Apply a patch
+
```

Comment:
I'd like to see more of an intro paragraph. 

The following example applies the X patch to ....